Week1/homework/app.js

```diff
@@ -3,9 +3,162 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'moby_dick',
+    'the_adventures_of_huckleberry_finn',
+    'alices_adventures_in_wonderland',
+    'charlie_and_the_chocolate_factory',
+    'wuthering_heights',
+    'war_and_peace',
+    'anna_karenina',
+    'animal_farm',
+    'a_tale_of_two_cities',
+    'the_alchemist'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  const Books = {
```

Comment:
In JavaScript, the variable naming convention is to make use of camelCasing as listed [here](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md). Quite similar to how you have named the `bookTitles` variable above.